devicetypes/smartthings/zwave-temp-humidity-sensor.src/zwave-temp-humidity-sensor.groovy

```diff
@@ -0,0 +1,206 @@
+			tileAttribute("device.temperature", key: "PRIMARY_CONTROL") {
+				attributeState "temperature", label: '${currentValue}°',
+					backgroundColors: [
+						[value: 31, color: "#153591"],
+						[value: 44, color: "#1e9cbb"],
+						[value: 59, color: "#90d2a7"],
+						[value: 74, color: "#44b621"],
+						[value: 84, color: "#f1d801"],
+						[value: 95, color: "#d04e00"],
+						[value: 96, color: "#bc2323"]
+					]
+			}
+		}
+		valueTile("humidity", "device.humidity", inactiveLabel: false, width: 2, height: 2) {
+			state "humidity", label: '${currentValue}% humidity', unit: ""
+		}
+		valueTile("dewPoint", "device.dewPoint", inactiveLabel: false, width: 2, height: 2) {
+			state "dewPoint", label: '${currentValue}° dewPoint', unit: ""
+		}
+		valueTile("battery", "device.battery", inactiveLabel: false, decoration: "flat", width: 2, height: 2) {
+			state "battery", label: '${currentValue}% battery', unit: ""
+		}
+		standardTile("refresh", "device.refresh", inactiveLabel: false, decoration: "flat", width: 2, height: 2) {
+			state "default", label: "", action: "refresh.refresh", icon: "st.secondary.refresh"
+		}
+
+		main "temperature", "humidity", "dewPoint"
+		details(["temperature", "humidity", "dewPoint", "battery"])
+	}
+}
+
+def updated() {
+	configure()
+}
+
+def installed() {
+	sendEvent(name: "checkInterval", value: 8 * 60 * 60 + 10 * 60, displayed: false, data: [protocol: "zwave", hubHardwareId: device.hub.hardwareID])
+}
+
+def configure() {
+	response([
+		secure(zwave.batteryV1.batteryGet()),
+		"delay 500",
+		secure(zwave.sensorMultilevelV5.sensorMultilevelGet(sensorType: 0x05)), // humidity
+		"delay 500",
+		secure(zwave.sensorMultilevelV5.sensorMultilevelGet(sensorType: 0x01)), // temperature
+		"delay 500",
+		secure(zwave.sensorMultilevelV5.sensorMultilevelGet(sensorType: 0x0B)), // dew point
+		"delay 500",
+		secure(zwave.wakeUpV2.wakeUpNoMoreInformation())
+	])
+}
+
+def parse(String description) {
+	def results = []
+
+	if (description.startsWith("Err")) {
+		results += createEvent(descriptionText: description, displayed: true)
+	} else {
+		def cmd = zwave.parse(description)
+		if (cmd) {
+			results += zwaveEvent(cmd)
+		}
+	}
+
+	log.debug "parse() result ${results.inspect()}"
+
+	return results
+}
+
+def zwaveEvent(physicalgraph.zwave.commands.notificationv3.NotificationReport cmd) {
+	log.debug "Event: ${cmd.event}, Notification type: ${cmd.notificationType}"
+	def result = []
+	def value
+	def description
+
+	if (cmd.notificationType == 0x10) {  // Mold Environment Detection
+		switch (cmd.event) {
+			case 0x00:
+				value = "clear"
+				description = "Mold environment not detected"
+				break
+			case 0x02:
+				value = "moldDetected"
+				description = "Mold environment detected"
+				break
+			default:
+				log.debug "Not handled event type: ${cmd.event}"
+				break
+		}
+
+	result = createEvent(name: "isMoldEnvironment", value: value, descriptionText: description, isStateChange: true, displayed: true)
```

Comment:
is there a custom capability for this or something? wouldn't it need to be referenced above?

Reply:
unfortunately not, but maybe we can use "Notification" capability in this case

Reply:
The Notification capability only provides a command, so it would be rather useless...

Sounds like we should propose a `Mold Detector` capability. I can create something for that today.

Reply:
@dkirker it would be great, when do You think it would be ready to implement? should we wait for the capability to be created along with metadata to push it further? 
@KKlimczukS

Reply:
@ZWozniakS I think it'd be a two week or so wait (minimum; but I don't think much review would be needed) to get the capability definition live. I think that might be fine. I'd prefer that. I'll reply back when I have the capability definition up.